code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -149,6 +149,12 @@
 
 /obj/item/bodypart/MiddleClick(mob/living/user, params)
 	var/obj/item/held_item = user.get_active_held_item()
+	var/meattype = /obj/item/reagent_containers/food/snacks/meat/steak
+	switch(species_icon)	//When we add tritons, they should butcher into fish.
+		if('icons/roguetown/mob/bodies/m/harpy.dmi', 'icons/roguetown/mob/bodies/f/harpy.dmi')					//Bird-type species, add medicators here once added.
+			meattype = /obj/item/reagent_containers/food/snacks/meat/poultry/cutlet
+		if('icons/roguetown/mob/bodies/m/mt_muscular.dmi', 'icons/roguetown/mob/bodies/f/ft_muscular.dmi')		//Half-orcs.
+			meattype = /obj/item/reagent_containers/food/snacks/meat/strange
```

Comment:
Please add a meat type variable to the limbs OR use species id instead of the icon